test/nyc-bin.js

```diff
@@ -561,10 +563,130 @@ describe('the nyc cli', function () {
+        var args = [bin, 'instrument', '--include', '**/subdir/input-dir/bad.js', './', './output']
+
+        var proc = spawn(process.execPath, args, {
+          cwd: fixturesCLI,
+          env: env
+        })
+
+        proc.on('close', function (code) {
+          code.should.equal(0)
+          var files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
+          files.length.should.equal(1)
+          var subdirExists = fs.existsSync(path.resolve(fixturesCLI, './output/subdir/input-dir'))
+          subdirExists.should.equal(true)
+          files = fs.readdirSync(path.resolve(fixturesCLI, './output/subdir/input-dir'))
+          files.should.include('bad.js')
+          files.should.not.include('index.js')
+          done()
+        })
+      })
+
+      it('allows a file to be excluded from an included directory', function (done) {
+        var args = [bin, 'instrument', '--exclude', '**/bad.js', '--include', '**/subdir/input-dir/**', './', './output']
+
+        var proc = spawn(process.execPath, args, {
+          cwd: fixturesCLI,
+          env: env
+        })
+
+        proc.on('close', function (code) {
+          code.should.equal(0)
+          var subdirExists = fs.existsSync(path.resolve(fixturesCLI, './output/subdir/input-dir'))
+          subdirExists.should.equal(true)
+          var files = fs.readdirSync(path.resolve(fixturesCLI, './output/subdir/input-dir'))
+          files.should.include('index.js')
+          files.should.not.include('bad.js')
+          done()
+        })
+      })
+
+      describe('es-modules', () => {
+        it('instruments files that should only instrument when es-modules is disabled', (done) => {
+          const args = [bin, 'instrument', '--es-modules', 'false', './not-strict.js', './output']
+
+          const proc = spawn(process.execPath, args, {
+            cwd: fixturesCLI,
+            env: env
+          })
+
+          proc.on('close', function (code) {
+            code.should.equal(0)
+            var subdirExists = fs.existsSync(path.resolve(fixturesCLI, './output'))
+            subdirExists.should.equal(true)
+            var files = fs.readdirSync(path.resolve(fixturesCLI, './output'))
+            files.should.include('not-strict.js')
+            done()
+          })
+        })
+
+        it('fails to instrument files that can\'t be instrumented when es-modules is enabled', (done) => {
+          const args = [bin, 'instrument', '--exit-on-error', 'true', './not-strict.js', './output']
+
+          const proc = spawn(process.execPath, args, {
+            cwd: fixturesCLI,
+            env: env
+          })
+
+          var stderr = ''
+          proc.stderr.on('data', function (chunk) {
+            stderr += chunk
+          })
+
+          proc.on('close', function (code) {
+            code.should.equal(1)
+            stdoutShouldEqual(stderr, `
+        Failed to instrument ./not-strict.js`)
```

Comment:
nit: Could you increase the indent so that the line with `Failed to instrument` is indented 2 spaces more than `stdoutShouldEqual`?

Reply:
Didn't realise that was running through a strip indent, will fix in the next commit.